scripts/local-setup/Procfile.web

```diff
@@ -0,0 +1,9 @@
+babbage: (cd ../../../babbage && ./run.sh)
+zebedee: (cd ../../../zebedee && ./run-reader.sh)
+sixteens: (cd ../../../zebedee && ./run.sh)
```

Comment:
You will want to add dp-design-system to this list as this gets used in the Go frontend web apps. Sixteens is used by Babbage and Florence (UI for the headless CMS, Zebedee)